spec/services/ccms/applicant_add_requestor_spec.rb

```diff
@@ -15,14 +31,14 @@ module CCMS
     describe '#transaction_request_id' do
       it 'returns the id based on current time' do
         Timecop.freeze(2019, 1, 2, 3, 4, 5.123456) do
-          requestor = described_class.new
+          requestor = described_class.new(applicant)
           expect(requestor.transaction_request_id).to start_with '20190102030405123456'
         end
       end
     end
 
     def expected_xml
-      File.read("#{File.dirname(__FILE__)}/data/expected_create_client_status_request.xml")
+      File.read("#{File.dirname(__FILE__)}/data/applicant_add_request.xml")
```

Comment:
There needs to be some consistency in how the data files are accessed. In other specs the file has been located relative to `Rails.root`. Here the files directory is being used (which be problematic if spec is moved).

Also, I think data files should be at spec root rather than nested within the specs. So move them to `spec/data/ccms`

As this is being used so much, I'd strongly recommend that this is wrapped in a method. Something like this:

Add a new file `spec/support/file_helpers.rb` with:
```ruby
def cms_data_from_file(filename)
  path = Rails.root.join 'spec/data/ccms', filename
  File.read path
end
```
Then this method can be used in specs. So for example, this method could be replaced with:
```ruby
let(:expected_xml) { cms_data_from_file 'applicant_add_request.xml' }
```